microsoft-edge/origin-trials/index.md

```diff
@@ -5,99 +5,20 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 02/02/2023
+ms.date: 10/24/2023
 ---
 # Use Origin Trials in Microsoft Edge
 
+You can use Origin Trials to try out experimental APIs on your own live site for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site might run code that uses experimental APIs.  To support the experimental APIs, users don't need to go to `edge://flags` on their own device to access the experimental APIs that your site uses.  Instead, you register the origin (for example, `https://www.contoso.com:443`) that you want to use the experimental feature on, and you'll then be sent a token that you add to the code of your website.<!-- todo: indicate whether the token is automatically sent immediately, or if there's a manual process that takes a day or so -->
```

Comment:
@amandabaker - todo: indicate whether the token is automatically sent immediately, or if there's a manual process that takes a day or so

Reply:
It's currently a manual process, but I'm not sure what we want to declare as an SLA for getting the token to the developer. @Reezaali WDYT?

Reply:
The surrounding Markdown content is Outdated.  Latest draft, in Files Changed tab of PR:
```
Instead, you register the origin (for example, `https://www.contoso.com:443`) that you want to 
use the experimental feature on, and 
you'll then be sent a token that you add to the code of your website.
Sending the token is not automatic and immediate, but is a manual process 
that can take a number of days.
<!-- todo: indicate whether the token is automatically sent immediately, 
or if there's a manual process that takes a day or so -->
```